platform/ios/INSTALL.md

```diff
@@ -52,8 +52,6 @@ bash "${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/Mapbox.framework/strip-fra
 
 #### Static framework
 
-If your application targets iOS 7.x, you’ll need to install the static framework instead:
```

Comment:
I struggled to come up with anything here — “if you can’t say anything nice...” 😉

Will revisit this tomorrow. Very open to detailed suggestions!